docker-compose.yml

```diff
@@ -19,6 +19,10 @@ services:
       - DB_PASSWORD=password
       - DB_NAME=chapter
       - DB_URL=db
+      - CHAPTER_EMAIL=ourEmail@placeholder.place
```

Comment:
`Env` variables will grow in the future, shouldn't we maintain separate `.env`

Reply:
for the most part, there's no functional difference between having a long list of env variables in the docker-compose file and a long list of env variables in `.env`.  With a separate .env file, you are requiring people to copy/rename a file before running it whereas this will be set up when you run `docker-compose up`.

Reply:
For the sake of simplicity I think that for now we should stick with env vars in docker-compose, but maybe later when we have a lot of them we could extract them.